docs/pages/database-access/guides/azure-postgres-mysql.mdx

```diff
@@ -222,19 +262,41 @@ Use the Azure `az` CLI utility to log in as the user that you set as the AD admi
 fetch the access token and use it as a password when connecting to the database:
 
 <Tabs>
+<TabItem label="PostgreSQL Flexible">
+
+```code
+$ az login -u ad@example.com
+$ export PGPASSWORD=`az account get-access-token --resource-type oss-rdbms --output tsv --query accessToken`
+$ psql "host=example.postgres.database.azure.com user=ad@example.com sslmode=require dbname=postgres"
```

Comment:
I originally suggested exporting the token, but let's update this so we don't export it:
```suggestion
$ TOKEN=`az account get-access-token --resource-type oss-rdbms --output tsv --query accessToken`
$ PGPASSWORD=$TOKEN psql "host=example.postgres.database.azure.com user=ad@example.com sslmode=require dbname=postgres"
```